### ISSUE TITLE
API change? Link urls broken

### ISSUE THREAD

**Issue by bnjmnrsh** [bnjmnrsh-projs/signal-v-noise#22]
It is likely related to #21, though not isolated to the sports route.  

The href attribute values in links to the NYT website are all missing. 

```html
<article class="news-item us loaded">
    <header><a href="" title="Chinese Spy Agency Rising to Challenge the C.I.A.">
            <img class="loading-bg" scrset="https://static01.nyt.com/images/2023/12/27/us/politics/27dc-china-techspy-02/27dc-china-techspy-02-superJumbo.jpg 1366w,
                    https://static01.nyt.com/images/2023/12/27/us/politics/27dc-china-techspy-02/27dc-china-techspy-02-threeByTwoSmallAt2X-v2.jpg 600w,
                    https://static01.nyt.com/images/2023/12/27/us/politics/27dc-china-techspy-02/27dc-china-techspy-02-thumbLarge.jpg 150w,
                        w"
                src="https://static01.nyt.com/images/2023/12/27/us/politics/27dc-china-techspy-02/27dc-china-techspy-02-threeByTwoSmallAt2X-v2.jpg"
                height="400" width="600"
                alt="Illustration by Chantal Jahchan; Photographs by Jim Wilson/The New York Times, Qilai Shen for The New York Times, Getty Images, and Associated Press"
                loading="lazy" role="presentation">
        </a></header>
    <section class="news-content">
        <header>
            <h3 class="hyphens"><a href="">Chinese Spy Agency Rising to Challenge the C.I.A.</a></h3>
            <p class="details"><a class="pill us" href="https://www.nytimes.com/section/us">us </a>
                <span class="byline">
                    By Edward Wong, Julian E. Barnes, Muyi Xiao and Chris Buckley
                </span>
            </p>
        </header>
        <div class="abstract">
            <p class="hyphens">The ambitious Ministry of State Security is deploying A.I. and other advanced technology
                to go toe-to-toe with the United States, even as the two nations try to pilfer each other’s trade
                secrets.
                <a href="" title="Chinese Spy Agency Rising to Challenge the C.I.A." class="read-more">[...more]</a>
            </p>
        </div>
    </section>
</article>
```

**Comment by bnjmnrsh** [comment]
This appears to be a bug in the API: the `short_url` param we have been using is empty.  Thankfully we still have the full `ulr` param, which works. 

<details><summary>JSON EXTRACT</summary>
<p>

```javascript
{
    "data": {
        "top_stories": {
            "status": "OK",
            "copyright": "Copyright (c) 2023 The New York Times Company. All Rights Reserved.",
            "section": "Style",
            "last_updated": "2023-12-25T21:25:15-05:00",
            "num_results": 25,
            "results": [
                {
                    "section": "travel",
                    "subsection": "",
                    "title": "The 8 Most-Read Travel Stories of 2023",
                    "abstract": "A golden retriever convention, flight etiquette rules and great walks from around the world: Here’s what readers loved this year.",
                    "url": "https://www.nytimes.com/2023/12/27/travel/travel-stories-readers-2023.html",
                    "uri": "nyt://article/d83e1382-e695-53be-a777-63c1f72a4139",
                    "byline": "By Tacey Rychter",
                    "item_type": "Article",
                    "updated_date": "2023-12-27T05:05:50-05:00",
                    "created_date": "2023-12-27T05:05:50-05:00",
                    "published_date": "2023-12-27T05:05:50-05:00",
                    "material_type_facet": "",
                    "kicker": "",
                    "des_facet": [
                        "Airlines and Airplanes",
                        "Travel and Vacations",
                        "Two Thousand Twenty Three",
                        "Content Type: Service"
                    ],
                    "org_facet": [],
                    "per_facet": [],
                    "geo_facet": [],
                    "multimedia": [
                        {
                            "url": "https://static01.nyt.com/images/2023/07/01/travel/travel-most-read-paris/22hours-paris-tjzf-superJumbo.jpg",
                            "format": "Super Jumbo",
                            "height": 1365,
                            "width": 2048,
                            "type": "image",
                            "subtype": "photo",
                            "caption": "A guide to 36 hours in Paris was — perhaps unsurprisingly — one of the most-read travel stories of 2023.",
                            "copyright": ""
                        },
                        {
                            "url": "https://static01.nyt.com/images/2023/07/01/travel/travel-most-read-paris/22hours-paris-tjzf-threeByTwoSmallAt2X.jpg",
                            "format": "threeByTwoSmallAt2X",
                            "height": 400,
                            "width": 600,
                            "type": "image",
                            "subtype": "photo",
                            "caption": "A guide to 36 hours in Paris was — perhaps unsurprisingly — one of the most-read travel stories of 2023.",
                            "copyright": ""
                        },
                        {
                            "url": "https://static01.nyt.com/images/2023/07/01/travel/travel-most-read-paris/22hours-paris-tjzf-thumbLarge.jpg",
                            "format": "Large Thumbnail",
                            "height": 150,
                            "width": 150,
                            "type": "image",
                            "subtype": "photo",
                            "caption": "A guide to 36 hours in Paris was — perhaps unsurprisingly — one of the most-read travel stories of 2023.",
                            "copyright": ""
                        }
                    ],
                    "short_url": ""
                }

...
``` 

</p>
</details>

**Comment by bnjmnrsh** [comment]
Closed by commit 37efaf5

**Comment by bnjmnrsh** [comment]
In response to an email to the developers at the NYT, @nyt-hughmandeville replied that the short URL existed for a time when social media sites were much more constrictive in the overall size of the post URL. Those limits are lifted, so the API has stopped supporting the internal URL shortening service.
